amundsen_application/static/js/pages/DashboardPage/index.tsx

```diff
@@ -100,6 +103,15 @@ export class DashboardPage extends React.Component<
     }
   }
 
+  onClick = (e) => {
+    const dashboardGroup = this.props.dashboard.group_name;
```

Comment:
We'll usually first destructure the props, like this:
```
const { dashboard, searchDashboardGroup } = this.props;

logClick(e, {...
   label: dashboard.group_name,
});
searchDashboardGroup(dashboardGroup);

Reply:
thanks @Golodhros , pr is updated